docs/community/oidc-integrations.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Community-Tested OIDC Integrations
+parent: Community
+nav_order: 4
+---
+
+# OIDC Integratins
+
+**Note** This is community-based content for which the core-maintainers cannot guarantee correctness. The parameters may change over time. If a parameter does not work as documented, please submit a PR to update the list.
+
+## Currently Tested Applications
+
+- GitLab (userinfo endpoint missing in an early implementation; now in peer review)
```

Comment:
HEAD now supports userinfo and GitLab was used to test it was working.

Reply:
👍🏼